### PR TITLE
Added Assets content to What Does OCS Do

### PR DESCRIPTION
**Pull request by osimarkrogers**
Hi Laureen,

This is my assets piece for the What Does OCS Do section - - if you'd like to take a look at this and enter any comments, I'll just count your review as Doc QA.  I've incorporated comments from Victor Zhang and it's been through peer review.

Thanks,

Mark

### CODE REVIEW

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 1]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
 
 The Trend feature converts stream data to a graphic view, which can reveal trends, high points, or trouble spots. Use Trend to select data streams in a namespace, specify a time range, and then render a graph of those data values.
 
 The following example shows stream data for two streams over a selected period of several days:
 
 ![OCS](images/how-does-ocs-work/trend.jpg)
 
+### Assets
+
+Assets are a framework that enables you to model data for different groupings of physical entities. An asset can consist of any number of data streams, and can be thought of as a digital twin for any physical entity.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:194]:
```suggestion
Assets are a digital twin .... that enables you to model data for different groupings of physical entities. An asset can consist of any number of data streams, and can be thought of as a digital twin for any physical entity. 
```

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 2]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
 
 The following example shows stream data for two streams over a selected period of several days:
 
 ![OCS](images/how-does-ocs-work/trend.jpg)
 
+### Assets
+
+Assets are a framework that enables you to model data for different groupings of physical entities. An asset can consist of any number of data streams, and can be thought of as a digital twin for any physical entity. 
+
+The assets in the example below each represent trucks in a fleet, with multiple data streams defined for each.  Any selected vehicle asset can be used to display  visualization of selected data streams for review and analysis, as in the following example:
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:196]:
```suggestion
The assets in the example below represent trucks in a fleet, with multiple data streams defined for each.  You can select an asset and display a trend of the data streams which can be used to identify and analyze problems.
```

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 3]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
+
+![OCS](images/how-does-ocs-work/assets-example.png)
+
+1. Select each of a truck's four suspension cylinders to display visualization of each cylinder's variation in pressure over time.
+2. Identify cylinders whose pressure is outside the accepted range to troubleshoot suspension issues.
+
+Assets are a great way to organize and contextualize data streams.  With PI to OCS transfers, for example, they provide a means of organizing multiple PI data tags on the OCS side into multiple streams under a single asset. For example, you could model an arena with streams measuring data for thermostats, ventilation equipment, lighting systems, and security as an asset based on a single entity with streams for multiple devices.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:203]:
```suggestion
Assets are a useful way to organize and contextualize data streams.  With PI to OCS data transfers, for example, you can organize multiple PI tags under a single asset. For example, you could create an asset with streams measuring data for thermostats, ventilation equipment, lighting systems, and security.
```

File: Content_Portal/Overview/what-does-ocs-do.md (hunk 1 above)
Comment (osimarkrogers) [reply Content_Portal/Overview/what-does-ocs-do.md:194]: Done.

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 4]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
+
+To define an asset, you can define an asset type to reuse for other assets, just as you would define a stream type for a stream.  While for stream types, the properties you define are data fields in a stream, the properties you define for asset types are streams to be included in an asset. However, assets can also be non-typed, and created on an ad hoc basis.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:205]:
```suggestion
Assets can be created on an ad hoc basis or they can be created from an asset type, a template for creating similar assets. 
```

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 5]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
+
+Users can also define asset rules, which bulk create or update assets according to matching patterns in stream names and metadata.  Asset rules are applied to existing applicable streams, and whenever a stream is created, updated, or deleted.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:207]: Mark, I am surprised Victor left this in. This feature is not yet in the product. It is part of the M4 release. I would remove this or at least verify with him that this feature is being exposed.

Comment (osimarkrogers) [reply Content_Portal/Overview/what-does-ocs-do.md:207]: Let me check with him....

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 6]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
 
+### Assets
+
+Assets are a digital twin of physical entities in the real world.  An asset can consist of data from one or more streams. 
+
+The assets in the example below represent trucks in a fleet, with multiple data streams defined for each.  You can select an asset and display a trend of the data streams which can be used to identify and analyze problems.
+
+![OCS](images/how-does-ocs-work/assets-example.png)
+
+1. Select each of a truck's four suspension cylinders to display visualization of each cylinder's variation in pressure over time.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:200]:
```suggestion
You can display the variation in the pressure of the suspension cylinders on each truck over time.
```

File: Content_Portal/Overview/what-does-ocs-do.md (hunk 5 above)
Comment (osimarkrogers) [reply Content_Portal/Overview/what-does-ocs-do.md:207]: Hi Laureen,
I just emailed you with Victor's response.

File: Content_Portal/Overview/what-does-ocs-do.md [hunk 7]
```diff
@@ -171,14 +171,41 @@ To create a stream view, designate a souce type that contains the desired proper
...
+2. Identify cylinders whose pressure is outside the accepted range to troubleshoot suspension issues.
```
Comment (lasato-OSI) [inline Content_Portal/Overview/what-does-ocs-do.md:201]: Remove this sentence.

Comment (osimarkrogers) [reply Content_Portal/Overview/what-does-ocs-do.md:201]: Right - - Done.

**Review by lasato-OSI — APPROVED** [review]

File: Content_Portal/Overview/what-does-ocs-do.md (hunk 5 above)
Comment (lasato-OSI) [reply Content_Portal/Overview/what-does-ocs-do.md]: Please add the caveat that this can be done using the API.

Comment (osimarkrogers) [reply Content_Portal/Overview/what-does-ocs-do.md]: Done

**Pull request merged**